src/pages/questionPage.js

```diff
@@ -1,77 +1,87 @@
+            currentAnswerElement = answerElement;
+            answersListElement
+                .querySelectorAll('.selected')
+                .forEach((element) => element.classList.remove('selected'));
+            answerElement.classList.add('selected');
+        });
+    }
 
-  document
-    .getElementById(NEXT_QUESTION_BUTTON_ID)
-    .addEventListener('click', nextQuestion);
+    document
+        .getElementById(NEXT_QUESTION_BUTTON_ID)
+        .addEventListener('click', nextQuestion);
 };
 
 const nextQuestion = () => {
-  const correctAnswer = quizData.questions[quizData.currentQuestionIndex].correct;
-  const addClass = quizData.currentQuestionAnswer === correctAnswer ? 'correct' : 'wrong';
-  const body = document.getElementById(USER_INTERFACE_ID);
-  if (quizData.currentQuestionAnswer===null) {
-    const alertElement = createAlertElement();
-    body.appendChild(alertElement);
-    quizData.currentQuestionIndex = quizData.currentQuestionIndex -1;
-  } else if(quizData.currentQuestionAnswer === correctAnswer ) {
-    currentAnswerElement.classList.remove('selected')
-    currentAnswerElement.classList.add(addClass)
-  } else {
-    currentAnswerElement.classList.remove('selected')
-    currentAnswerElement.classList.add(addClass)
-  }
-  quizData.currentQuestionAnswer = null
-  quizData.currentQuestionIndex = quizData.currentQuestionIndex + 1;
-  document.getElementById(NEXT_QUESTION_BUTTON_ID).removeEventListener('click', nextQuestion);
+    const correctAnswer =
+        quizData.questions[quizData.currentQuestionIndex].correct;
+    const addClass =
+        quizData.currentQuestionAnswer === correctAnswer ? 'correct' : 'wrong';
+    const body = document.getElementById(USER_INTERFACE_ID);
+    const soundCorrect = new Audio('../../public/sounds/sound_correct.mp3');
+    const soundWrong = new Audio('../../public/sounds/sound_wrong.mp3');
+
+    if (quizData.currentQuestionAnswer === null) {
+        const alertElement = createAlertElement();
+        body.appendChild(alertElement);
+        quizData.currentQuestionIndex = quizData.currentQuestionIndex - 1;
+    } else if (quizData.currentQuestionAnswer === correctAnswer) {
+        currentAnswerElement.classList.remove('selected');
+        currentAnswerElement.classList.add(addClass);
+        soundCorrect.play();
+    } else {
+        currentAnswerElement.classList.remove('selected');
+        currentAnswerElement.classList.add(addClass);
+        soundWrong.play();
+    }
+    quizData.currentQuestionAnswer = null;
+    quizData.currentQuestionIndex = quizData.currentQuestionIndex + 1;
+    document
+        .getElementById(NEXT_QUESTION_BUTTON_ID)
+        .removeEventListener('click', nextQuestion);
 
-  setTimeout(() => {
-    initQuestionPage();
-    currentAnswerElement.classList.remove(addClass)
-  }, 1500);
+    setTimeout(() => {
+        initQuestionPage();
+        currentAnswerElement.classList.remove(addClass);
+        soundCorrect.pause();
```

Comment:
I would just let the sounds play, sounds a bit strange that they are cut off now. The wrong answer sound is quite long though, maybe we want to shorten it somehow, but that's up to you.